.github/workflows/ci.yml

```diff
@@ -28,37 +28,37 @@ jobs:
 
     strategy:
       matrix:
-        eslint: [6.x]
-        node: [14.x]
+        eslint: [7]
+        node: [14]
         os: [ubuntu-latest]
         include:
           # On other platforms
-          - eslint: 6.x
-            node: 14.x
+          - eslint: 7
+            node: 14
             os: windows-latest
-          - eslint: 6.x
-            node: 14.x
+          - eslint: 7
+            node: 14
             os: macos-latest
           # On old Node.js versions
-          - eslint: 6.x
-            node: 12.x
+          - eslint: 7
+            node: 12
             os: ubuntu-latest
-          - eslint: 6.x
-            node: 10.x
-            os: ubuntu-latest
-          - eslint: 6.x
-            node: 8.x
+          - eslint: 7
+            node: 10
             os: ubuntu-latest
           # On old ESLint versions
-          - eslint: 5.x
-            node: 14.x
+          - eslint: 6
+            node: 14
+            os: ubuntu-latest
+          - eslint: 5
+            node: 14
             os: ubuntu-latest
-          - eslint: 4.x
-            node: 14.x
+          - eslint: 4
+            node: 14
             os: ubuntu-latest
           # On the minimum supported ESLint/Node.js version
-          - eslint: 4.x
-            node: "8.0.0"
+          - eslint: 4
+            node: 8.10.0
```

Comment:
Please don't change the minimum supposed Node.js version.

Reply:
`8.10.0` is the minimum that's inside `package.json`'s `engines`

Reply:
Oh, you're right. I'm sorry.